.github/workflows/main.yaml

```diff
@@ -39,12 +17,7 @@ jobs:
         with:
           java-version: 17
           distribution: 'temurin'
-      - name: Set up cache
-        uses: actions/cache@v3
-        with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
-          restore-keys: ${{ runner.os }}-maven-
+          cache: 'maven'
```

Comment:
Hvorfor gjøres denne endringen? Er det GitHub som har fått innebygd støtte for caching via `setup-java`-action eller er det noe annet som har endret seg?

Reply:
Fant svaret: [`setup-java`-action har caching innebygd](https://github.com/actions/setup-java#caching-packages-dependencies).